src/backtrack/ReadMaze.java

```diff
@@ -0,0 +1,75 @@
+package backtrack;
+
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ReadMaze {
+    private Path FilePath;
+    private int [][] startPos = {{0,0}};
```

Comment:
IMO, it is better to have a class Position than a double array for this
`` 
class Position {
            private int x;
            private int y;
            public Position(int x, int y) {
                this.x = x;
                this.y = y;
            }
}...
``
To me, it is easy to read startPosition.getX, startPosition.getY than startPos[0][0], startPos[0][1]
Also, it is less error prone, you can for example do a mistake startPos[0][-1] or startPos[1][1]